### ISSUE TITLE
Duplicate names but different Covers 

### ISSUE THREAD

**Issue by eliekh05** [aldostools/webMAN-MOD#1069]
@aldostools @Evilnat and others

i am having issue with my webman the issue is that when i install multiman and webman to copy games using multiman then when i want to refresh webman games using the button that is available in webman setup on xmb and the refresh xmb using the button that is available in webman setup on xmb i am having the duplicate names but different covers like FIFA 19 twice but different covers like one FIFA 19 cover and other is like GTA San Andreas cover but named FIFA 19 or whatever is it webman or multiman issue if it is fixable please tell me what to do and fix it in the code

System: PS3HEN
Exploit: HEN
Firmware: HFW 4.91

**Comment by aldostools** [comment]
Without more information about your setup & files it's hard to give a diagnostic.

First you have to post the screen captures of your games in USB, HDD and XMB showing the issue. And configuration in /setup.ps3

webMAN MOD scans all drives checked in /setup.ps3 and removes duplicate entires in the USB drives if games have the exact file or folder name in the internal HDD.

What you mention does not make me sense unless you have the games actually duplicated with local covers or wrong title id in the file name or in PARAM.SFO.

**Comment by eliekh05** [comment]
> Without more information about your setup & files it's hard to give a diagnostic.
> 
> 
> 
> First you have to post the screen captures of your games in USB, HDD and XMB showing the issue. And configuration in /setup.ps3
> 
> 
> 
> webMAN MOD scans all drives checked in /setup.ps3 and removes duplicate entires in the USB drives if games have the exact file or folder name in the internal HDD.
> 
> 
> 
> What you mention does not make me sense unless you have the games actually duplicated with local covers or wrong title id in the file name or in PARAM.SFO.

when i uninstall the webman and multiman and reinstall webman it fixes it and if i want to copy iso using multiman because webman crashes when copying over 5gb after copying i go to xmb use the buttons that i told you and the issue will be back it is not related to usb or anything that you specified

**Comment by aldostools** [comment]
Try using prepISO instead the buttons to refresh the games. Report if that fixes your issue.

**Comment by eliekh05** [comment]
> Try using prepISO instead the buttons to refresh the games. Report if that fixes your issue.

if it doesn't fix it then what should i do tell me which files should i send you and i will

![image](https://github.com/aldostools/webMAN-MOD/assets/67870780/fb8cbfa2-8a3c-4023-b25e-f9d9467d7f72)

![image](https://github.com/aldostools/webMAN-MOD/assets/67870780/50864886-0d8e-4b03-8aab-5f43bc2d773a)

**Comment by aldostools** [comment]
Upload /dev_hdd0/xmlhost/game_plugin/mygames.xml

And make capture of the files your /dev_hdd0/PS3ISO using IRISMAN or multiMAN file manager.

Or uncheck PARAM.SFO in /setup.ps3 to show the file name instead of the Title name in PARAM.SFO

**Comment by eliekh05** [comment]
> Upload /dev_hdd0/xmlhost/game_plugin/mygames.xml
> 
> And make capture of the files your /dev_hdd0/PS3ISO using IRISMAN or multiMAN file manager.
> 
> Or uncheck PARAM.SFO in /setup.ps3 to show the file name instead of the Title name in PARAM.SFO


unzip the file:
[mygames.xml.zip](https://github.com/aldostools/webMAN-MOD/files/15139653/mygames.xml.zip)

and i did uncheck for param.sfo

**Comment by aldostools** [comment]
It seems that the PARAM.SFO of `Need for Speed SHIFT.ISO` is corrupted and showing FIFA 19
The icon and file name of the ISO are correct. You can try an older version of webMAN MOD to check if it's some bug in the latest build.

````
  <T key="0002" include="inc">
   <P key="icon"><>/dev_hdd0/PS3ISO/Need for Speed SHIFT.png</></P>
   <P key="title"><>FIFA 19</></P>
   <P key="module_action"><>/mount_ps3/dev_hdd0/PS3ISO/Need+for+Speed+SHIFT.ISO</></P>
   <P key="info"><>hdd0/PS3ISO</></P>
  </T>
  <T key="0003" include="inc">
   <P key="icon"><>/dev_hdd0/PS3ISO/FIFA 19.png</></P>
   <P key="title"><>FIFA 19</></P>
   <P key="module_action"><>/mount_ps3/dev_hdd0/PS3ISO/FIFA+19.ISO</></P>
   <P key="info"><>hdd0/PS3ISO</></P>
  </T>

**Comment by eliekh05** [comment]
> It seems that the PARAM.SFO of `Need for Speed SHIFT.ISO` is corrupted and showing FIFA 19 The icon and file name of the ISO are correct. You can try an older version of webMAN MOD to check if it's some bug in the latest build.
> 
> ```
>   <T key="0002" include="inc">
>    <P key="icon"><>/dev_hdd0/PS3ISO/Need for Speed SHIFT.png</></P>
>    <P key="title"><>FIFA 19</></P>
>    <P key="module_action"><>/mount_ps3/dev_hdd0/PS3ISO/Need+for+Speed+SHIFT.ISO</></P>
>    <P key="info"><>hdd0/PS3ISO</></P>
>   </T>
>   <T key="0003" include="inc">
>    <P key="icon"><>/dev_hdd0/PS3ISO/FIFA 19.png</></P>
>    <P key="title"><>FIFA 19</></P>
>    <P key="module_action"><>/mount_ps3/dev_hdd0/PS3ISO/FIFA+19.ISO</></P>
>    <P key="info"><>hdd0/PS3ISO</></P>
>   </T>
> ```


[mygames.xml.zip](https://github.com/aldostools/webMAN-MOD/files/15139697/mygames.xml.zip)
i uploaded the latest vrsion of .xml file unzip and can you fix it from your side as official everyone gets the fix if not tell me which version and i will do it

**Comment by aldostools** [comment]
No, I cannot fix your issue unless I know the exact cause the issue. 
You should check if the PARAM.SFO of your games are corrupted or if the title is being obtained from some external PARAM.SFO

````
  <T key="0005" include="inc">
   <P key="icon"><>/dev_hdd0/PS3ISO/LEGO Batman 2 DC Super Heroes.png</></P>
   <P key="title"><>Grand Theft Auto IV</></P>
   <P key="module_action"><>/mount_ps3/dev_hdd0/PS3ISO/LEGO+Batman+2+DC+Super+Heroes.ISO</></P>
   <P key="info"><>hdd0/PS3ISO</></P>
  </T>

**Comment by aldostools** [comment]
You can try any previous version. e.g. 1.47.38 or 1.47.44

**Comment by eliekh05** [comment]
> No, I cannot fix your issue unless I know the exact cause the issue. You should check if the PARAM.SFO of your games are corrupted or if the title is being obtained from some external PARAM.SFO
> 
> ```
>   <T key="0005" include="inc">
>    <P key="icon"><>/dev_hdd0/PS3ISO/LEGO Batman 2 DC Super Heroes.png</></P>
>    <P key="title"><>Grand Theft Auto IV</></P>
>    <P key="module_action"><>/mount_ps3/dev_hdd0/PS3ISO/LEGO+Batman+2+DC+Super+Heroes.ISO</></P>
>    <P key="info"><>hdd0/PS3ISO</></P>
>   </T>
> ```

is there a way to reset param.sfo and if i reinstall webman without the multiman everything is normal

**Comment by eliekh05** [comment]
> You can try any previous version. e.g. 1.47.38 or 1.47.44

ok

**Comment by aldostools** [comment]
> is there a way to reset param.sfo and if i reinstall webman without the multiman everything is normal

No, unless you rebuild the ISO with the correct PARAM.SFO

If you unchecked PARAM.SFO in /setup.ps3 and re-scan the games the title should show the file name of the ISO instead of the title name found in PARAM.SFO

**Comment by eliekh05** [comment]
> > is there a way to reset param.sfo and if i reinstall webman without the multiman everything is normal
> 
> No, unless you rebuild the ISO with the correct PARAM.SFO
> 
> If you unchecked PARAM.SFO in /setup.ps3 and re-scan the games the title should show the file name of the ISO instead of the title name found in PARAM.SFO

thanks it is fixed by uninstalling the multiman and webman and reinstalling latest version of webman how can i prevent it in the future

**Comment by eliekh05** [comment]
> > > is there a way to reset param.sfo and if i reinstall webman without the multiman everything is normal
> > 
> > 
> > No, unless you rebuild the ISO with the correct PARAM.SFO
> > If you unchecked PARAM.SFO in /setup.ps3 and re-scan the games the title should show the file name of the ISO instead of the title name found in PARAM.SFO
> 
> thanks it is fixed by uninstalling the multiman and webman and reinstalling latest version of webman how can i prevent it in the future

how can i prevent it in the future

**Comment by aldostools** [comment]
If it's fixed, it seems there is something wrong in the version of multiMAN that you're using or there's something wrong in your games and you think it's fixed due the uncheck of PARAM.SFO in /setup.ps3.

**Comment by eliekh05** [comment]
> If it's fixed, it seems there is something wrong in the version of multiMAN that you're using or there's something wrong in your games and you think it's fixed due the uncheck of PARAM.SFO in /setup.ps3.

@aldostools maybe multiman is not supported by hen only cfw or it has not been updated for a long time because without it it is working perfectly the games are checked and updated and configured (trophies, data, and dlc) after each copy to internal
